sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -364,14 +364,18 @@ class HttpLoggingPolicy(SansIOHTTPPolicy):
         "Transfer-Encoding",
         "User-Agent",
     ])
+
+    # Deprecated
+    DEFAULT_HEADERS_WHITELIST = DEFAULT_HEADERS_ALLOWLIST
+
     REDACTED_PLACEHOLDER = "REDACTED"
 
     def __init__(self, logger=None, **kwargs):  # pylint: disable=unused-argument
         self.logger = logger or logging.getLogger(
             "azure.core.pipeline.policies.http_logging_policy"
         )
         self.allowed_query_params = set()
-        self.allowed_header_names = set(HttpLoggingPolicy.DEFAULT_HEADERS_WHITELIST)
+        self.allowed_header_names = set(self.DEFAULT_HEADERS_ALLOWLIST)
```

Comment:
This fixes a bug in ARMHttpLoggingPolicy. Previously the allowed_header_names would be set to HttpLoggingPolicy.DEFAULT_HEADERS_ALLOWLIST instead of ARMHttpLoggingPolicy.DEFAULT_HEADERS_ALLOWLIST